vendor/bat-native-ads/src/bat/ads/internal/notifications.h

```diff
@@ -29,8 +29,9 @@ class Notifications {
   bool Get(const std::string& id, NotificationInfo* info) const;
 
   void Add(const NotificationInfo& info);
+  void Remove(); //removes the front item
 
-  bool Remove(const std::string& id);
+  bool Remove(const std::string& id); //removes an item by id
```

Comment:
Please change function name to RemoveWithId as then we do not need the comment and the code is more readable

Reply:
renamed as `PushBack/PopFront`